enclave/core/sgx/snmalloc/snmalloc_wrapper.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/sgxtypes.h>
+#include "../../oe_alloc_thread.h"
+#include "../../oe_nodebug_alloc.h"
+
+namespace snmalloc
+{
+__thread void* allocator_local;
```

Comment:
Thanks for using a thread local as opposed to td_t. It allows us to explore removing td_t (#2554) without worrying that we may break clients like CCF.